app/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = yeoman.generators.Base.extend({
     this.directory('.', '.');
   },
 
+  generateYoRc: function() {
+    this.log('Generating .yo-rc.json');
+    this.config.save();
+  },
+
```

Comment:
Sure, I will pull this feature in a separate PR. I have no idea how to unit test this, but we can discuss it over there instead. ;)